.github/workflows/pkg-docker.yml

```diff
@@ -21,7 +21,7 @@ jobs:
         id: meta
         uses: docker/metadata-action@v3
         with:
-          images: nlnetlabs/routinator
+          images: ${{ secrets.DOCKER_HUB_ID }}/routinator
```

Comment:
FYI using the DOCKER_HUB_ID secret this isn't a good idea here as the Docker Hub ID can be that of a CI user, e.g. `nlnetlabsci`, rather than the organization name that is part of the image tag.

Reply:
Thanks!  This makes sense - I didn't think of that situation.  Great pickup.